PermissionsScrapper/PermissionsScrapper/ProtectedApiCallHelper.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ------------------------------------------------------------------------------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace PermissionsScrapper
+{
+    /// <summary>
+    /// Helper class to call a protected API and returns the result of the call
+    /// </summary>
+    public class ProtectedApiCallHelper
+    {
+        public ProtectedApiCallHelper(HttpClient httpClient)
+        {
+            HttpClient = httpClient;
+        }
+
+        protected HttpClient HttpClient { get; private set; }
+
+        /// <summary>
+        /// Calls a protected Web API
+        /// </summary>
+        /// <param name="webApiUrl">Url of the Web API to call</param>
+        /// <param name="accessToken">Access token used as a bearer security token to call the Web API</param>
+        /// <returns>The response of the call to the Web API</returns>
+        public async Task<string> CallWebApiAsync(string webApiUrl, string accessToken)
```

Comment:
why not just a static function and pass in the httpClient directly since there's no other value in instantiating the Helper class.

Reply:
Resolved